src/webxdc.rs

```diff
@@ -20,6 +20,23 @@ use zip::ZipArchive;
 pub const WEBXDC_SUFFIX: &str = "xdc";
 const WEBXDC_DEFAULT_ICON: &str = "__webxdc__/default-icon.png";
 
+/// Defines the maximal size in bytes of an .xdc file that can be sent.
+///
+/// We introduce a limit to force developer to create small .xpc
+/// to save user's traffic and disk space for a better ux.
+///
+/// The 100K limit should also let .xdc pass worse-quality auto-download filters
+/// which are usually 160K incl. base64 overhead.
+///
+/// The limit is also an experiment to see how small we can go;
+/// it is planned to raise that limit as needed in subsequent versions.
+pub(crate) const WEBXDC_SENDING_LIMIT: usize = 102400;
+
+/// Be more tolerant for .xdc sizes on receiving -
+/// might be, the senders version uses already a larger limit
+/// and not showing the .xpc on some devices would be even worse ux.
+const WEBXDC_RECEIVING_LIMIT: usize = 1048576;
```

Comment:
Is it intentional that this is 1mb? (like 10 time the sending limit?)

Reply:
tbh, I don't think that limiting the size helps UX more the opposite, but well besides that, I don't think it is necessary to limit the receiving part, only the sending, I plan myself to have no limits (or make it higher in my fork, and it will be a pity if users using official DC can't even receive them, probably they will then move to the fork to play bigger games :/

Reply:
ok, maybe not limiting at all would be an idea, but we don't want to incentivise breaking our limits with forks and spacing users with supper big apps, but then again we have message size download limits anyway so users can already protect against that scenario

Reply:
the receiving limit is pretty high for reasons @adbenitez pointed out - it is worse ux when an app does not work just because of this - and we aim to increase allowed sizes on sending, is pretty probable.